solcjs

```diff
@@ -89,8 +89,17 @@ var output = solc.compile({ sources: sources }, argv.optimize ? 1 : 0);
 if (!output) {
   abort('No output from compiler');
 } else if (output['errors']) {
+  function isWarning (message) {
+    return message.match(/^(.*:[0-9]*:[0-9]* )?Warning: /)
+  }
+
   for (var error in output['errors']) {
-    console.error(output['errors'][error]);
+    var message = output['errors'][error]
+    if (isWarning(message)) {
+      console.log(message)
```

Comment:
Does solcjs produce any other output on stdout? My feeling is that the data should go to stdout and anything user-facing through the "side-channel" stderr. But if we do not have output on stdout anyway (I didn't see anything), then this is fine.

Reply:
Data goes into files.